Python-3.7.7: module curses not installed

I am using a script that depends on the npyscreen module, which depends on the curses module.
When trying to import the curses module an exception is raised:
```
Python 3.7.7 (default, Aug 18 2020, 20:42:48) 
[GCC 4.9.3 20150311 (prerelease)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import curses
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/python3/lib/python3.7/curses/__init__.py", line 13, in <module>
    from _curses import *
ModuleNotFoundError: No module named '_curses'
```
So it seems that the _curses module is not properly installed although it is usually part of the standard library (the script was working fine on the previously packaged python version v3.5.6).

### Setup
_Package Name:_ Python3
_Package Version:_ 3.7.7-12

_NAS Model:_ DS415Play
_NAS Architecture:_ 
_DSM version:_ 6.2

### Expected behavior
The curses module should be installed by default.

### Actual behavior
Exception is raised because the _curses module is not installed (it was installed on the previous python3 version 3.5.6).

### Steps to reproduce
_1._ start the python3 interpreter with /usr/local/python3/bin/python3
_2._ import curses

#### Package log
`/usr/local/python3/var/python3_install.log`
```
Fri Aug 28 06:41:22 +08 2020
===> Step preupgrade. USER= GROUP= SHARE_PATH=
Fri Aug 28 06:41:22 +08 2020
===> Step preuninst. USER= GROUP= SHARE_PATH=
Fri Aug 28 06:41:27 +08 2020
===> Step postuninst. USER= GROUP= SHARE_PATH=
Fri Aug 28 06:41:29 +08 2020
===> Step preinst. USER= GROUP= SHARE_PATH=
Fri Aug 28 06:41:32 +08 2020
===> Step postinst. USER= GROUP= SHARE_PATH=
Invoke service_postinst
Looking in links: /volume1/@appstore/python3/share/wheelhouse
Processing /volume1/@appstore/python3/share/wheelhouse/MarkupSafe-1.1.1-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/PyYAML-5.3-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/SQLAlchemy-1.3.13-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/appdirs-1.4.3-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/distlib-0.3.0-py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/filelock-3.0.12-py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/idna-2.9-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/immutables-0.11-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/importlib_metadata-1.5.0-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/importlib_resources-1.4.0-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/ipaddress-1.0.23-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/lxml-4.5.0-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/msgpack_python-0.5.6-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/olefile-0.46-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/psutil-5.7.0-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/pyOpenSSL-19.1.0-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/pyasn1-0.4.8-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/pycurl-7.43.0.5-cp37-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/ruamel.yaml-0.16.10-py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/six-1.14.0-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/virtualenv-20.0.5-py2.py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/zipp-3.1.0-py3-none-any.whl
Processing /volume1/@appstore/python3/share/wheelhouse/zope.interface-4.7.1-cp37-none-any.whl
Installing collected packages: MarkupSafe, PyYAML, SQLAlchemy, appdirs, distlib, filelock, idna, immutables, importlib-metadata, importlib-resources, ipaddress, lxml, msgpack-python, olefile, psutil, pyOpenSSL, pyasn1, pycurl, ruamel.yaml, six, virtualenv, zipp, zope.interface
Successfully installed MarkupSafe-1.1.1 PyYAML-5.3 SQLAlchemy-1.3.13 appdirs-1.4.3 distlib-0.3.0 filelock-3.0.12 idna-2.9 immutables-0.11 importlib-metadata-1.5.0 importlib-resources-1.4.0 ipaddress-1.0.23 lxml-4.5.0 msgpack-python-0.5.6 olefile-0.46 psutil-5.7.0 pyOpenSSL-19.1.0 pyasn1-0.4.8 pycurl-7.43.0.5 ruamel.yaml-0.16.10 six-1.14.0 virtualenv-20.0.5 zipp-3.1.0 zope.interface-4.7.1
Python 3.7.7

System installed modules:
appdirs==1.4.3
bcrypt==3.1.7
cffi==1.14.0
cryptography==2.9.2
distlib==0.3.0
filelock==3.0.12
idna==2.9
immutables==0.11
importlib-metadata==1.5.0
importlib-resources==1.4.0
ipaddress==1.0.23
lxml==4.5.0
MarkupSafe==1.1.1
msgpack-python==0.5.6
olefile==0.46
psutil==5.7.0
pyasn1==0.4.8
pycparser==2.19
pycrypto==2.6.1
pycurl==7.43.0.5
pyOpenSSL==19.1.0
PyYAML==5.3
ruamel.yaml==0.16.10
six==1.14.0
SQLAlchemy==1.3.13
virtualenv==20.0.5
zipp==3.1.0
zope.interface==4.7.1
Fri Aug 28 06:42:44 +08 2020
===> Step postupgrade. USER= GROUP= SHARE_PATH=
```
#### Other logs
_E.g. `/var/log/messages` or `/var/log/synopkg.log`_
```
Insert log here
```

Just checked and it seems that indeed the `_curses` module is not appropriately cross compiled.

```
2020-08-16T22:52:29.0429855Z building '_curses' extension
2020-08-16T22:52:29.0434933Z /github/workspace/toolchains/syno-evansport-6.1/work/i686-pc-linux-gnu/bin/i686-pc-linux-gnu-gcc -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/github/workspace/toolchains/syno-evansport-6.1/work/i686-pc-linux-gnu/i686-pc-linux-gnu/sys-root/usr/include -I/github/workspace/spk/python3/work-evansport-6.1/install//var/packages/python3/target/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -L /github/workspace/spk/python3/work-evansport-6.1/install/var/packages/python3/target/lib -I /github/workspace/spk/python3/work-evansport-6.1/install/var/packages/python3/target/include -I/github/workspace/toolchains/syno-evansport-6.1/work/i686-pc-linux-gnu/i686-pc-linux-gnu/sys-root/usr/include -I/github/workspace/spk/python3/work-evansport-6.1/install//var/packages/python3/target/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -L /github/workspace/spk/python3/work-evansport-6.1/install/var/packages/python3/target/lib -I /github/workspace/spk/python3/work-evansport-6.1/install/var/packages/python3/target/include -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-missing-field-initializers -Werror=implicit-function-declaration -DHAVE_NCURSESW=1 -I./Include -I. -I/github/workspace/toolchains/syno-evansport-6.1/work/i686-pc-linux-gnu/i686-pc-linux-gnu/sys-root/usr/include -I/github/workspace/spk/python3/work-evansport-6.1/install//var/packages/python3/target/include -I/github/workspace/toolchains/syno-evansport-6.1/work/i686-pc-linux-gnu/bin/../i686-pc-linux-gnu/sys-root/usr/include -I/github/workspace/spk/python3/work-evansport-6.1/Python-3.7.7/Include -I/github/workspace/spk/python3/work-evansport-6.1/Python-3.7.7 -c /github/workspace/spk/python3/work-evansport-6.1/Python-3.7.7/Modules/_cursesmodule.c -o build/temp.linux-i686-3.7/github/workspace/spk/python3/work-evansport-6.1/Python-3.7.7/Modules/_cursesmodule.o
2020-08-16T22:52:29.1197860Z /github/workspace/spk/python3/work-evansport-6.1/Python-3.7.7/Modules/_cursesmodule.c: In function ‘curses_window_addch_impl’:
2020-08-16T22:52:29.1198776Z /github/workspace/spk/python3/work-evansport-6.1/Python-3.7.7/Modules/_cursesmodule.c:629:9: error: implicit declaration of function ‘setcchar’ [-Werror=implicit-function-declaration]
2020-08-16T22:52:29.1199111Z          setcchar(&wcval, wstr, attr, attr_to_color_pair(attr), NULL);
2020-08-16T22:52:29.1199285Z          ^
[...]
2020-08-16T22:52:54.0432961Z Failed to build these modules:
2020-08-16T22:52:54.0433262Z _curses                                                        
2020-08-16T22:52:54.0433348Z 
```

--- a/cross/python3/Makefile
+++ b/cross/python3/Makefile
@@ -8,7 +8,7 @@ PKG_DIST_NAME = Python-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = http://www.python.org/ftp/python/$(PKG_VERS)
 PKG_DIR = Python-$(PKG_VERS)
 
-DEPENDS  = cross/zlib cross/openssl cross/sqlite cross/readline cross/ncurses cross/bzip2 cross/xz
+DEPENDS  = cross/zlib cross/openssl cross/sqlite cross/readline cross/ncursesw cross/bzip2 cross/xz
 # required for uuid module to be available
 DEPENDS += cross/libuuid
 # required because libffi is no longer bundled with python

--- a/spk/python3/Makefile
+++ b/spk/python3/Makefile
@@ -1,7 +1,7 @@
 SPK_NAME = python3
 SPK_SHORT_VERS = 3.7
 SPK_VERS = $(SPK_SHORT_VERS).7
-SPK_REV = 12
+SPK_REV = 13
 SPK_ICON = src/python3.png
 
 DEPENDS  = cross/busybox cross/$(SPK_NAME)
@@ -21,7 +21,7 @@ DESCRIPTION_SPN = Lenguaje de programación Python.
 RELOAD_UI = yes
 STARTABLE = no
 DISPLAY_NAME = Python3
-CHANGELOG = "1. Update to Python 3.7.7"
+CHANGELOG = "1. Fixed curses native module"
 
 HOMEPAGE = http://www.python.org
 LICENSE  = PSF

